masonite/snippets/auth/controllers/PasswordController.py

```diff
@@ -0,0 +1,49 @@
+"""A PasswordController Module."""
+
+import uuid
+
+from masonite import env
+from masonite import Mail
+from masonite import Session
+from masonite.auth import Auth
+from masonite.helpers import password as bcrypt_password
+from masonite.request import Request
+from masonite.view import View
+
+from app.User import User
```

Comment:
Don't import the user model directly. it could be moved around. Instead you should import the auth config model. Like:

```
from config.auth import AUTH

AUTH['model] # == User model or whatever model they choose to be the authentication model
```